meson.build

```diff
@@ -0,0 +1,24 @@
+#  Copyright © 2022 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+project('lc3', 'c',
+	version: '0.1',
+	license: 'Apache-2.0',
+	meson_version : '>= 0.47.0')
+
+subdir('src')
+
+if get_option('tools').enabled()
```

Comment:
Why not just use a boolean option here instead of a feature? It's a lot simpler, and currently `-Dtools=auto` means the same as `-Dtools=disabled`, so making it be `-Dtools=true` / `=false` is a lot clearer.